hip-clang.docker

```diff
@@ -45,9 +44,6 @@ RUN ldconfig
 # Workaround broken miopen cmake files
 RUN sed -i 's,;/usr/lib/x86_64-linux-gnu/librt.so,,g' /opt/rocm/lib/cmake/miopen/miopen-targets.cmake
 
-# Workaround for broken hip-clang
-# RUN sed -i -e '/if(WIN32)/, /endif()/d' /opt/rocm/lib/cmake/hip/hip-config.cmake && sed -i -n 5p /opt/rocm/lib/cmake/hip/hip-config.cmake
```

Comment:
I think the workaround can be removed from sles as well.

Reply:
removed